features/controller_specs/cookies.feature

```diff
@@ -0,0 +1,36 @@
+Feature: Cookies
+
+  There is different ways to test cookies in controller's tests be we recommend
+  to use `cookies` as below.
```

Comment:
I think this needs rewording, theres a few grammatical issues. How about:
```
There are different ways to make assertions on cookies from controller specs, but we recommend using the `cookies` method as set out below.
```

Reply:
Thanks. It is much better with your proposal. Changed with https://github.com/rspec/rspec-rails/pull/2127/commits/54104d8ea3ce6c4f75be17518f34593799930371